sys/include/embUnit/AssertImpl.h

```diff
@@ -46,8 +46,8 @@ void assertImplementationCStr(const char *expected,const char *actual, long line
 
 #define TEST_ASSERT_EQUAL_STRING(expected_, actual_) \
     do { \
-        __typeof__(expected_) ____expected__ = expected_; \
-        __typeof__(actual_) ____actual__ = actual_; \
+        const char *____expected__ = expected_; \
+        const char *____actual__ = actual_; \
```

Comment:
Why is this change required?

Reply:
Something didn't work without it, but I cannot remember what.  :angel: